documentation/3.3/content/userguide/aks/_index.md

```diff
@@ -133,7 +134,24 @@ If you want to upload existing keystores, select **Upload existing KeyStores** f
 | Trust KeyStore Data file(.jks,.p12) | Upload a custom trust keystore data file by doing the following: {{< line_break >}} 1. Click on the file icon. {{< line_break >}} 2. Navigate to the folder where the identity keystore file resides, and select the file. {{< line_break >}} 3. Click Open. |
 | Password | Enter the password for the custom trust keystore. |
 | Confirm password | Re-enter the value of the preceding field. |
-| The Identity KeyStore type (JKS,PKCS12) | Select the type of custom trust keystore. The supported values are JKS and PKCS12. |
+| The Trust KeyStore type (JKS,PKCS12) | Select the type of custom trust keystore. The supported values are JKS and PKCS12. |
+
+If you want to use keystores that are stored in Azure Key Vault, select **Use KeyStores stored in Azure Key Vault** for the option **How would you like to provide required configuration**, and enter the values for the fields listed in the following table.
+
+##### Use KeyStores stored in Azure Key Vault
+
+| Field | Description |
+|-------|-------------|
+| Resource group name in current subscription containing the Key Vault | Enter the name of the Resource Group containing the Key Vault that stores the SSL certificate and the data required for WebLogic SSL termination. |
+| Name of the Azure Key Vault containing secrets for the TLS/SSL certificate | Enter the name of the Azure Key Vault that stores the SSL certificate and the data required for WebLogic SSL termination. |
+| The name of the secret in the specified Key Vault whose value is the Identity KeyStore Data | Enter the name of the Azure Key Vault secret that holds the value of the identity keystore data. Follow [Store the TLS/SSL certificate in the Key Vault](#store-the-tlsssl-certificate-in-the-key-vault) to upload certificate to Azure Key Vault. |
```

Comment:
upload certificate -> upload the certificate (please correct all instances; I found 13.)

Reply:
See https://github.com/oracle/weblogic-kubernetes-operator/pull/3174/commits/83051c4dc5feb07e74d5e8ac0e3ddcd05d60b3ec